src/vtbackend/Terminal.h

```diff
@@ -966,7 +988,10 @@ class Terminal
     void resetStatusLineDefinition();
 
     TabsInfo guiTabsInfoForStatusLine() const noexcept { return _guiTabInfoForStatusLine; }
-    void setGuiTabInfoForStatusLine(TabsInfo info) { _guiTabInfoForStatusLine = info; }
+    void setGuiTabInfoForStatusLine(TabsInfo&& info) { _guiTabInfoForStatusLine = std::move(info); }
+
+    TabsNamingMode getTabsNamingMode() const noexcept { return _settings.tabNamingMode; }
+    void setTabName();
```

Comment:
can we maybe figure out a better name of it? Because `setTabName` sounds like you're setting a tab name, and you're usually also passing the name.